src/main/java/org/reactome/release/cosmicupdate/COSMICIdentifierUpdater.java

```diff
@@ -0,0 +1,194 @@
+package org.reactome.release.cosmicupdate;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceDisplayNameGenerator;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+/**
+ * Updates a COSMIC Identifier.
+ * Implements <code>Comparable</code> to make sorting easier for reporting purposes. Does *not* override the <code>equals</code> method.
+ * @author sshorser
+ *
+ */
+public class COSMICIdentifierUpdater implements Comparable<COSMICIdentifierUpdater>
+{
+	private static final Logger logger = LogManager.getLogger();
+	private static GKInstance updateWithNewCOSV;
+	private static GKInstance updatePrependCOSM;
+	private String identifier;
+	private Long dbID;
```

Comment:
No need to change it but something that has bitten me in the past is comparing `Long` objects to see if they are equal with `==` and getting `false` when the values were the same because it checked reference equality (with `Long` objects you need to use `.equals()` for value equality).  

I always use primitive `long` where I can so it's not an issue.